llmfoundry/models/layers/blocks.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         device: Optional[str] = None,
         no_bias: bool = False,
         use_pad_tok_in_ffn: bool = True,
+        extra_args_to_exclude_in_attn_class: Optional[Set[str]] = None,
```

Comment:
instead of passing in an arg here and then union with the default, can you make this a property on the `MPTBlock` class?

Reply:
Good idea. Done.